src/getOperands.js

```diff
@@ -26,7 +26,10 @@
  * @return {string} - operands
  */
 function getOperands(str) {
-  // write code here
+  const regularOfFloat = /-?\d+[.]?[\d+]?/g;
+  const operandsArray = str.match(regularOfFloat);
```

Comment:
```suggestion
  const operands = str.match(regularOfFloat);
```
не дописывайте явное array